aptos-move/framework/aptos-framework/sources/timestamp.move

```diff
@@ -35,7 +36,7 @@ module aptos_framework::timestamp {
         timestamp: u64
     ) acquires CurrentTimeMicroseconds {
         // Can only be invoked by AptosVM signer.
-        system_addresses::assert_vm(account);
+        system_addresses::is_reserved_address(address_of(account));
```

Comment:
plan to run block_prologue_ext as 0x1 instead of 0x0 so for a few resources (e.g. BlockRandomness, ValidatorSetChangeLocked) we can avoid on-chain state initialization and move_from/move_to during `block_prologue_ext`.